pages/linux/lvremove.md

```diff
@@ -0,0 +1,11 @@
+# lvremove
+
+> Remove one or more logical volumes.
```

Comment:
We can add the link to the `lvm2` site.
```suggestion
> Remove one or more logical volumes.
> More information: <https://sourceware.org/lvm2/>.
```

Reply:
Hi @navarroaxel, thanks for your comments. I modified it. Cheers!